ocs/defaults.py

```diff
@@ -8,3 +8,6 @@
 ROOK_CLUSTER_NAMESPACE = 'openshift-storage'
 KUBECONFIG_LOCATION = 'auth/kubeconfig'  # relative from cluster_dir
 CLUSTER_NAME = f"ocs-ci-cluster-{getuser()}"
+VOLUME_DELETED = 'persistentvolume "{volume_name}" deleted'
```

Comment:
I feel like this deserve better place than defaults, same below.

Reply:
I agree,
Any proposed file name?

Reply:
It seems like something that can just be handled within the script? You're really just building out a string with the volume name, no need to have a central place for something like this.

Reply:
agree with Coady that this can be just part of the module where are you using this